### PR TITLE
Day 5

### PR DESCRIPTION
**Pull request by josejorgers**

### CODE REVIEW

File: tutorial/guias/Paso_a_paso_DIA_5.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+>**Nota :pen:** En la línea anterior usamos lo que se conoce como *string interpolation*. Cuando anteponemos la letra `f` antes de un string en Python, podemos incluir dentro del string código Python. En este caso queremos escribir *Score: x* donde *x* es el número que representa el score. Como dicho score está en la variable `score`, podemos escribir el string como: `f"Score: {score}"`. Al anteponer una `f` y encerrar la variable score entre llaves, obtenemos el resultado deseado.
+>**Nota :pen:** Toda la documentación sobre Pygame la puedes encontrar [aqui](https://www.pygame.org/docs/) (en inglés).
```
Comment (albexl) [inline tutorial/guias/Paso_a_paso_DIA_5.md:151]: This note is formatted a little bit weirdly, I think. Should it be a blank line between both notes? I don't know. Take a look at how it looks in the preview and consider adding a blank line or something so the second note is more noticeable.

File: tutorial/guias/Paso_a_paso_DIA_5.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+```python
+for pos in snake_body:
+    pygame.draw.rect(
+        game_window, snake_color, pygame.Rect(pos[0], pos[1], 10, 10)
+    )
+```
+
+Por cada posición en la lista que representa el cuerpo de la serpiente, usamos el campo `draw` de Pygame para dibujar. Específicamente vamos a representar cada parte del cuerpo como un pequeño cuadrado de 10 píxeles de lado. Para ello usamos la función `rect` para pintar rectángulos, le pasamos la ventana del juego, el color, y un objeto de tipo `Rect` que está definido en Pygame. A este último objeto le debemos pasar la posición de la esquina superior izquierda, el ancho y el largo del rectángulo. De esta forma queda pintado nuestro pequeño cuadradito con el color deseado :snake:.
+
+Por último, y de manera análoga, pintamos un pequeño cuadrado para representar la fruta. La única diferencia es que este cuadrado tendrá el color definido para la fruta.
```
Comment (albexl) [inline tutorial/guias/Paso_a_paso_DIA_5.md:199]: Maybe add the code snippet for this example here

**Review by albexl — CHANGES_REQUESTED** [review]
Minor stuff to check. Looks great overall.

File: tutorial/guias/Paso_a_paso_DIA_5.md (hunk 1 above)
Comment (josejorgers) [reply tutorial/guias/Paso_a_paso_DIA_5.md:151]: True. I removed the blank line because VS Code was complaining about it, but it's best to keep it

**Pull request merged**